utils/crypto/bls/signer/rpcsigner/client.go

```diff
@@ -21,7 +24,24 @@ type Client struct {
 	pk     *bls.PublicKey
 }
 
-func NewClient(ctx context.Context, conn *grpc.ClientConn) (*Client, error) {
+func NewClient(ctx context.Context, rpcSignerURL string) (*Client, error) {
+	// TODO: figure out the best parameters here given the target block-time
+	opts := grpc.WithConnectParams(grpc.ConnectParams{
+		Backoff: backoff.DefaultConfig,
+	})
+
+	// the rpc-signer client should call a proxy server (on the same machine) that forwards
+	// the request to the actual signer instead of relying on tls-credentials
+	conn, err := grpc.NewClient(rpcSignerURL, opts, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		return nil, fmt.Errorf("couldn't create rpc signer client: %w", err)
```

Comment:
nit: I generally avoid special characters where possible in logs errors since you usually have to escape them when grepping through logs. Can we make this something like `could not` or `failed to` or anything along those lines? Our old code doesn't follow this pattern but I think we should avoid it in new code we write